src/api/app/models/bs_request.rb

```diff
@@ -1067,6 +1067,11 @@ def required_checks
     target_project_objects.pluck(:required_checks).flatten.uniq
   end
 
+  # TODO: rename the 'notify_parameters' method instead of wrapping it like this.
+  def event_parameters
```

Comment:
@saraycp not sure, I dont see the ```event_parameters``` methods used anywhere in the bs_request model, do you wanted to use it in line 782?

Reply:
besides this the PR LGTM! Great work :)

Reply:
Sorry forget about it, overlooked something